bigquery/.gitignore

```diff
@@ -0,0 +1 @@
+docs/_build
```

Comment:
I think this is already being ignored in the top-level .gitignore?  https://github.com/googleapis/google-cloud-python/blob/cb1382e1b3bb606a8d527d988ccadec1ba9c29ec/.gitignore#L52

Reply:
No it wasn't, (git kept wanting to add the folder before I added the gitignore) but I can add it there to the top-level if you'd prefer?

Reply:
We'll be splitting the repo eventually anyway.  It's fine to keep it here. :+1: